rest-api/tools/import_organizations.py

```diff
@@ -108,6 +108,42 @@ def _insert_entity(self, entity, existing_map, session, dry_run):
     super(HPOImporter, self)._insert_entity(entity, existing_map, session, dry_run)
 
 
+  def delete_sql_statement(self, session, hpo_id_list):
+    str_list = ','.join([str(i) for i in hpo_id_list])
+    sql = """
+          DELETE FROM hpo
+          WHERE hpo_id IN ({str_list})
+          AND NOT EXISTS(
+          SELECT * FROM participant p, participant_history ph, site s, organization org
+          WHERE p.hpo_id IN ({str_list})
+          OR
+          ph.hpo_id in ({str_list})
+          OR
+          s.hpo_id in ({str_list})
+          OR
+          org.hpo_id in ({str_list})
+          )
+          """.format(str_list=str_list)
+
+    session.execute(sql)
+
+  def _cleanup_old_entities(self, session, row_list):
+    hpo_dao = HPODao()
+    hpo_group_list_from_sheet = []
+    existing_hpos = set(hpo.name for hpo in hpo_dao.get_all())
+    [hpo_group_list_from_sheet.append(row[HPO_AWARDEE_ID_COLUMN].upper()) for row in row_list]
```

Comment:
I think it would be simpler to remove the [] assignment above and write this as:

hpo_group_list_from_sheet = [row[HPO_AWARDEE_ID_COLUMN].upper() for row in row_list]

Reply:
done